.github/workflows/build.yml

```diff
@@ -21,3 +21,8 @@ jobs:
       - name: Run Gradle Check
         run: |
           ./gradlew check
+      - name: Upload Coverage Report
+        uses: codecov/codecov-action@v3.1.0
+        with:
+          file: ./build/reports/jacoco/test/jacocoTestReport.xml
+          flags: plugin
```

Comment:
Do we really need a flag here?

Reply:
I following the step which from the [guideline](https://github.com/opensearch-project/opensearch-plugins/blob/main/TESTING.md#code-coverage-collecting). Here is the [example](https://github.com/opensearch-project/anomaly-detection/blob/main/.github/workflows/CI.yml#L103) from the guideline. As I know for the flags, flags is allow us to isolate and categorize code coverage report for different test. If we not need to categorize, I can delete that.